src/graph_world/beam/generator_config_sampler.py

```diff
@@ -88,6 +88,18 @@ def SampleConfig(self, marginal=False):
             param_value = spec.default_val
       # If the param val is still None, give it a random value.
       if param_value is None:
-        param_value = spec.sampler_fn(spec)
+        # Allow parameters not being sampled to remain in output as null
```

Comment:
I think with my suggestions in src/graph_world/nodeclassification/generator_wrappers.py, there should be no sampler spec dictionary items that do not have a sampler fn. Regardless, please make sure we follow a design where this chunk is not needed.